framework/src/modules/dpos_v2/utils.ts

```diff
@@ -29,6 +23,8 @@ import {
 	WAIT_TIME_VOTE,
 } from './constants';
 
+const { q } = math;
```

Comment:
btw here it should be `q96`

Reply:
Resolves [here](https://github.com/LiskHQ/lisk-sdk/pull/7777/commits/1d240c9f2d3f2e7840e9b0fcdf5fd633e76127ec)